projects/demo/src/modules/app/app.routes.ts

```diff
@@ -951,14 +951,125 @@ export const ROUTES = [
             title: `Tree`,
         },
     },
+    // EDITOR
     {
         path: `components/editor`,
         loadChildren: async () =>
-            (await import(`../components/editor/editor.module`)).ExampleTuiEditorModule,
+            (await import(`../components/editor/getting-started/editor.module`))
+                .ExampleTuiEditorModule,
         data: {
-            title: `Editor`,
+            title: `Editor[Getting started]`,
```

Comment:
Let's change format to something more human, like `Editor — Getting started`. Same for the rest of Editor pages.

Reply:
fixed